build.psm1

```diff
@@ -1139,7 +1146,7 @@ function Start-PSPester {
     }
 
     # Autoload (in subprocess) temporary modules used in our tests
-    $command += '$env:PSModulePath = '+"'$TestModulePath$TestModulePathSeparator'" + '+$($env:PSModulePath);'
+    $command += '$env:PSModulePath = '+"'$TestModulePath$TestModulePathSeparator$script:TestModuleDirPath$TestModulePathSeparator'" + '+$env:PSModulePath;'
```

Comment:
I'm afraid of such expressions. Maybe use Join-Path before the line?

Reply:
`Join-Path` won't work since we want `;` and not `\` (or `:` and not `/` on *nix). But I'll break this expression up a bit